app/models/auction_query.rb

```diff
@@ -98,7 +98,7 @@ def public_find(id)
       .find(id)
   end
 
-  def my_bids(user_id)
+  def user_has_bid(user_id)
```

Comment:
It actually returns a collection of auctions where the user has placed a bid, so I struggled with the name a bit